### ISSUE TITLE
New line rendered as a box on Linux

### ISSUE THREAD

**Issue by ArthurCueio** [linebender/druid#1545]
On a multiline `TextBox`:

![Screenshot_2021-01-21_19-17-13](https://user-images.githubusercontent.com/25566621/105419457-8039a300-5c1d-11eb-9a56-8dc7b15913f0.png)

Code to reproduce:

```rust
use druid::widget::TextBox;
use druid::{AppLauncher, PlatformError, Widget, WindowDesc};

fn main() -> Result<(), PlatformError> {
    let main_window = WindowDesc::new(ui_builder);
    let data = String::new();
    AppLauncher::with_window(main_window)
        .use_simple_logger()
        .launch(data)
}

fn ui_builder() -> impl Widget<String> {
    TextBox::multiline()
}
```
This happens on the latest commit of master as of the moment I'm writing this (953bd0b) on both X11 and GTK backends.

**Comment by Zarenor** [comment]
Thanks for reporting this! I think the issue is known - I know a contributor is currently on text enhancements on Linux, and their latest screenshots appear to have this bug resolved. Hopefully they can finish their work and PR it soon!

**Comment by cmyr** [comment]
yes, this will be closed when https://github.com/linebender/piet/pull/389 is merged, and we get a new piet release.

**Comment by cmyr** [comment]
this should be fixed with #1677
